processing/src/main/java/io/druid/query/QueryContextKeys.java

```diff
@@ -24,4 +24,5 @@
   public static final String PRIORITY = "priority";
   public static final String TIMEOUT = "timeout";
   public static final String CHUNK_PERIOD = "chunkPeriod";
+  public static final String DIST_TARGET_SOURCE = "distTargetSource";
```

Comment:
Suggested "DISTRIBUTION_TARGET_SOURCE", I don't see why it should be abbreviated. Same for the String value.